docs/section19/19-5.md

```diff
@@ -0,0 +1,34 @@
+# Section 19 - Rule 19-5      
+**Schema Version:** 0.0.22  
+**Mandatory Rule:** True      
+**Rule ID:** 19-5       
+**Rule Description:** Unmet load hours for the proposed design shall not exceed 300 (of the 8760 hours simulated).   
+**Rule Assertion:** Options are Pass/Fail     
+**Appendix G Section:** G3.1.2.3        
+**90.1 Section Reference:** None  
+
+**Data Lookup:** None  
+
+**Evaluation Context:** Each Proposed Design RulesetModelInstance  
+
+**Applicability Checks:**  
+
+1. Applies only to the proposed design rule set model instance.  
+
+**Function Calls:**  None
+
+## Rule Logic:   
+**Applicability Check 1:**  
+- For each RulesetModelInstance in ASHRAE229.ruleset_model_instance : `for RulesetModelInstance in ASHRAE229.ruleset_model_instance:`  
+    - Check if the ruleset model instance is the proposed model instance: `If RulesetModelInstance.RulesetModelOptions2019ASHRAE901.ruleset_model_type = "PROPOSED":`  
+        - Get the heating unmet load hours for the rule set model instance: `unmet_load_hours_heating = RulesetModelInstance.OutputInstance.unmet_load_hours_heating`  
+        - Get the cooling unmet load hours for the rule set model instance: `unmet_load_hours_cooling = RulesetModelInstance.OutputInstance.unmet_load_hours_cooling`  
```

Comment:
There is no output instance yet - we will need to wait for the latest schema that indicates the correct jsonpath to retrieve the `unmet_load_hours_` results from output format. This needs to be confirmed with @JasonGlazer

Reply:
This is the output path to unmet_load_hours...

`RulesetModelInstance.output.OutputInstance.unmet_load_hours_heating`
`RulesetModelInstance.output.OutputInstance.unmet_load_hours_cooling`